src/components/Logo.astro

```diff
@@ -1,19 +1,29 @@
 ---
 import type { HTMLAttributes } from 'astro/types'
+import classNames from 'classnames'
 
 type Props = HTMLAttributes<'a'> & {
   secondaryStyles?: boolean
 }
 
-const { class: classNames, ...props } = Astro.props
```

Comment:
You could've left this and just added, class as an argument to the `classNames()`. In order to support customization for `Logo.astro` as well via tailwind classes.